datajunction-server/datajunction_server/api/helpers.py

```diff
@@ -233,30 +233,26 @@ def get_query(  # pylint: disable=too-many-arguments
 def find_bound_dimensions(
```

Comment:
Do you remember why we call it "bound" in this function name?

Reply:
@agorajek Yeah, I think that used to be an internal name for this, but then for all the user-facing parts we switched it to "required" dimensions. I do prefer required dimensions overall and might just switch them all to use that terminology for consistency.